rubocop/rubocop.yml

```diff
@@ -1,3 +1,5 @@
+require: rubocop-rails
```

Comment:
What cops in this file are in `rubocop-rails`? I would expect to see this set up in `rubocop/rubocop_rails.yml`, and seems like we should move any cops that are part of that into there as well.

That way, any apps using this that aren't dependent on rails aren't using those checks.

Reply:
Other than `Rails/NotNullColumn`, which you point out in your PR description. Anyway, do you think that movement/organization makes sense?

Reply:
I added it to `rubocop.yml` only because the `Rails/NotNullColumn` cop was in the file. I do think it makes more sense to move both that cop and `require: rubocop-rails` to `rubocop_rails.yml`.

It does make this change slightly backwards incompatible, which should probably be reflected in the release number. But it shouldn't cause any problems, I don't think anyone using this gem is inheriting directly from `rubocop/rubocop.yml` then enabling Rails in their local `.rubocop.yml`.

Reply:
I completely agree that this is a breaking change. We're still technically < 1.0, so I think we're ok that way, but I would release this as a full point version higher (so, going from 0.7.0 to 0.8.0, for example).